test/utils/test_display.jl

```diff
@@ -0,0 +1,40 @@
+include("../../src/utils/text_display.jl")
+
+@testset "text_display" begin
+    @testset "get_scientific_factors" begin
+        get_scientific_factors = ADRIA.get_scientific_factors
+
+        @testset "Edgecases" begin
+            @test get_scientific_factors(0.0) == (0.0, 0)
```

Comment:
Meta comment - no action required as such.

```julia
julia> get_scientific_factors(1e6)  # or 1_000_000.0
(10.0, 5)

julia> 10e5
1.0e6
```

Although they are the same, 1e6 is more understandable to me compared to 10e5 but I suppose it's hard to generalize these down further.

Reply:
So, that was a typo, my idea was to name this file `test/utils/text_display.jl`. Do you think this is ok?

Reply:
I think you replied to the wrong comment, but okay with me.

Reply:
On the above: a bit hacky, but what if we parse the value determined by your function to get Julia to auto-infer the simplified value?

```
julia> parse(Float64, "10e5")
1.0e6

julia> string(parse(Float64, "10e5"))
"1.0e6"
```

Reply:
Hm... let me think about it for a while...

Reply:
@ConnectedSystems  I think I solved the problem (and I've added some more tests, thanks for catching that). Care to take a look?

Reply:
Ooo, how about the opposite sign?

```julia
julia> -0.000001
-1.0e-6

julia> to_scientific(-0.000001)
"-10.0E-7"
```

Reply:
Cool, going to fix that too, thanks!

Reply:
@ConnectedSystems Done!